atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/watch/NoOpLockWatchManager.java

```diff
@@ -16,23 +16,28 @@
 
 package com.palantir.atlasdb.keyvalue.api.watch;
 
+import com.palantir.atlasdb.keyvalue.api.cache.NoOpTransactionScopedCache;
+import com.palantir.atlasdb.keyvalue.api.cache.TransactionScopedCache;
 import com.palantir.lock.watch.CommitUpdate;
-import com.palantir.lock.watch.LockWatchEventCache;
+import com.palantir.lock.watch.LockWatchCache;
+import com.palantir.lock.watch.LockWatchCacheImpl;
 import com.palantir.lock.watch.LockWatchReferences;
 import com.palantir.lock.watch.LockWatchVersion;
 import com.palantir.lock.watch.TransactionsLockWatchUpdate;
 import java.util.Optional;
 import java.util.Set;
 
-public final class NoOpLockWatchManager extends LockWatchManager {
-    private final LockWatchEventCache eventCache;
+public final class NoOpLockWatchManager extends LockWatchManagerInternal {
+    private final LockWatchCache cache;
+    private final TransactionScopedCache transactionScopedCache;
```

Comment:
I expect I'll find out why this is happening - the no op value cache should happily hand out no op transaction caches, so this feels weird.

Reply:
(Update: didn't work out why you did this - why not just create no op caches on the fly?)

Reply:
We can inline, sure